### PR TITLE
Fix a typo.

### PR DESCRIPTION
**Pull request by ismailarilik**
'a index' should be 'an index'.

### CODE REVIEW

File: docs/reference/docs/delete.asciidoc [hunk 1]
```diff
@@ -89,7 +89,7 @@ setting the routing parameter.
 
 Note that deleting a parent document does not automatically delete its
 children. One way of deleting all child documents given a parent's id is
-to use the <<docs-delete-by-query,Delete By Query API>> to perform a
+to use the <<docs-delete-by-query,Delete By Query API>> to perform an
```
Comment (cbuescher) [inline docs/reference/docs/delete.asciidoc]: Looking at the context, I think it should rather be something like "to perform a delete for..."

**Review by cbuescher — CHANGES_REQUESTED** [review]
Hi @ismailarilik, thanks for opening this PR. I left a comment with some suggestions.

**Comment by ismailarilik** [comment]
I have made the change you mentioned. But I also think this statement in the same sentence can be enhanced:

> which is in the format parent_type#parent_id

Users can write a query like that after this statement:

```
{
  parent_id: 'parent_type#parent_id'
}
```

Of course, this can be right. But I proofed this solution and I think it is much more elegant:

```
{
  parent_id: {
    type: parent_type,
    id: parent_id
  }
}
```

**Review by cbuescher — APPROVED** [review]
Thanks, I think the change is good as it is

**Pull request merged**